tests/test_shell.py

```diff
@@ -86,3 +86,33 @@ def test_init_flowproject_template(self, template):
         self.call(f"python -m flow init -t {template}".split())
         assert str(signac.get_project()) == "project"
         assert os.path.exists("project.py")
+
+    def test_template_create_base(self):
```

Comment:
Add some tests to verify that the `--help` message is printed.

In most of the tests here, if something is written to stdout you should verify that by capturing the output, like in this sketch:
```python
out = self.call("python...")
assert "Template created!" in out
```